test/Spec.hs

```diff
@@ -96,7 +96,7 @@ sOUFFLE_DIRS = ["souffle0", -- large Doop example
                 "souffle1", "souffle2", "souffle3", "souffle4", "souffle5", "souffle6",
                  -- "souffle7", -- uses a recursive type
                  "souffle9", "souffle10", 
-                 "souffle11", "souffle12", "souffle13", "souffle14", "souffle15", "souffle16", "souffle17", "souffle18"]
+                 "souffle11", "souffle12", "souffle13", "souffle14", "souffle15", "souffle16", "souffle17", "souffle18", "souffle20"]
```

Comment:
where is souffle19?

Reply:
I am afraid that it was already measing, I did not change a thing, I only add souffle20.

Reply:
I have added that; will be part of a future PR.